app/initializers/authenticators.js

```diff
@@ -1,107 +1,172 @@
 import config from '../config/environment';
 import { purecloudEnvironment } from '../utils/purecloud-environment';
-import { purecloudEnvironmentTld } from '../utils/purecloud-environment';
-import platformClient from 'platformClient';
+import Account from '../utils/account';
 
 export default {
 	name: 'authenticators',
-	featureTogglesToQuery: ['archDevToolsScripting'],
-	environment: 'environment',
-	/**
-	 * Creates the query string to get all the feature toggles that Dev-center tools cares about.
-	 * Creates this string based on the featureTogglesToQuery array
-	 * @return {string} - the query string
-	 * @private
-	 */
-	_createQueryString() {
-		var queryString = '';
-		this.featureTogglesToQuery.forEach(
-			function (toggle) {
-				queryString = queryString + 'feature=' + toggle + '&';
-			}.bind(this)
-		);
-		return queryString;
-	},
-
-	authenticate(client, oauthConfig, state, application) {
-		let returnedState;
+	accounts: 'accounts',
 
-		// Authenticate
-		client
-			.loginImplicitGrant(oauthConfig.clientId, oauthConfig.redirect, { state: state })
-			.then((data) => {
-				window.localStorage.setItem(this.environment, client.environment);
-				// Store returned state
-				returnedState = data.state;
+	removeAccount(account) {
+		let accountObj;
+		let storage = window.localStorage;
+		let storedAccounts = storage.getItem(this.accounts);
+		if (storedAccounts === 'null' || !storedAccounts) {
+		} else {
+			accountObj = JSON.parse(StoredAccounts);
+			for (let i = 0; i < accountObj.accounts.length; i++) {
+				if (accountObj.accounts[i].token === account.token) {
+					accountObj.accounts.splice(i, 1);
+				}
+			}
+			storage.setItem(this.accounts, JSON.stringify(accountObj));
+		}
+	},
 
-				// Makes call to platform api to get the feature toggles for the user that is logged in.
-				return $.ajax({
-					url: `https://apps.${client.environment}/platform/api/v2/featuretoggles?` + this._createQueryString(),
-					type: 'GET',
-					dataType: 'json',
-					headers: {
-						Authorization: `bearer ${data.accessToken}`,
-					},
-				});
-			})
-			.then((data) => {
-				if (!data) {
-					return;
+	removeInitializedDuplicates() {
+		let accountObj;
+		let storage = window.localStorage;
+		let StoredAccounts = storage.getItem('initialized');
+		if (StoredAccounts === 'null' || !StoredAccounts) {
```

Comment:
Maybe just have the code check for StoredAccounts !== 'null' || StoredAccounts instead of having an empty if block. Same goes for removeAccountDuplicates